### PR TITLE
actions. added migration action from queue to cdr

### PR DESCRIPTION
**Pull request by edospadoni**
nethesis/dev#5907

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethvoice-report-conf [hunk 1]
```diff
@@ -42,3 +42,14 @@ scl enable rh-php56 /var/lib/asterisk/bin/retrieve_conf
 
 # fix bad user_authorizations.json owner
 /usr/bin/chown asterisk:asterisk /opt/nethvoice-report/api/user_authorizations.json
+
+# check if migration is necessary to fix tables and searches
+MIGRATE_RES=$(/usr/bin/mysql --defaults-file=/root/.my.cnf asteriskcdrdb -e "SHOW TABLES LIKE 'dashboard_cdr_%'")
+if [[ "x$MIGRATE_RES" == "x" ]]; then
+	# migration is needed: drop data_call table with bad fields. overnight will be recreated
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethvoice-report-conf:49]:
```suggestion
	# migration is needed: drop data_call table with bad fields. The table will be recreated overnight.
```

**Review by gsanchietti — APPROVED** [review]
I'd suggest a cosmetic fix on current comment.

**Comment by edospadoni** [comment]
> I'd suggest a cosmetic fix on current comment.

fixed

**Comment by gsanchietti** [comment]
You can just squash and merge ;)

**Pull request merged**